Editor/AGS.Editor/DataFileWriter.cs

```diff
@@ -1392,7 +1397,8 @@ public static bool SaveThisGameToFile(string fileName, Game game, CompileMessage
             //   foreach (cap in caps)
             //       FilePutString(cap.Name);
             //
-            WriteGameSetupStructBase(writer, game);
+            long ext_off_pos; // position to write extensions offset to
```

Comment:
long behaves different between Windows 64-bit and other platforms when they are 64-bit, can it be other type?

Reply:
Yes, it's C#, and I've been using exact type as returned from Stream.Position:
https://learn.microsoft.com/en-us/dotnet/api/system.io.stream.position?view=net-8.0